content/q/gradle-quickstart.sh

```diff
@@ -46,6 +46,9 @@ defaultFlinkVersion="${1:-1.17.0}"
 # passes the scala version prefixed with a _, e.g.: _2.12
 scalaBinaryVersionFromCmdArg="${2/_/}"
 defaultScalaBinaryVersion="${scalaBinaryVersionFromCmdArg:-2.12}"
+defaultJavaVersion="1.8"
```

Comment:
It would be nice to have all the variable in a single place. We could extend the list that's already present in `docs/config.toml` and use a basic awk command to extract the corresponding values, e.g. `awk -F'"' '/FlinkStableVersion = "/{print $2}' docs/config.toml`. WDYT?

```bash
function extract_parameter() {
  if [[ "$#" != 1 ]]; then 
    echo "Fatal error: No parameter or too many parameter passed: $@"
    exit 1; 
  fi
  awk -F'"' '/'$1' = "/{print $2}' docs/config.toml
}
```